### ISSUE TITLE
More info on tap

### ISSUE THREAD

**Issue by ldsmovil** [kirbo/ha-lovelace-elapsed-time-card#5]
![image](https://user-images.githubusercontent.com/73847117/141089913-d9d8c153-a5f4-4fc6-9461-ff77cd6c891c.png)

i have this card con turn on/off tv and your card yo get the time is turned on, but i would like to tap over the icon to show the graph when the sensor has been stopped to view the exact time the tv has been turned off. Thanks

**Comment by Kirbo** [comment]
Good call! That's something I was thinking at some point but decided to leave it out since I didn't personally need it 😅
I'll add it soon-ish!
